docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-product-offer-feature-walkthrough/product-offer-storage.md

```diff
@@ -14,20 +14,21 @@ For details about how to use and configure Redis as a key-value storage, see [Us
 
 The following modules are used for the Product Offer storage:
 
-| MODULE | DESCRIPTION |
-| -------------------- | ---------- |
-| [MerchantProductOfferStorage](https://github.com/spryker/merchant-product-offer-storage) | Provides entity listeners and operates data  caching according to the entity changes |
-| [MerchantProductOfferStorageExtension](https://github.com/spryker/merchant-product-offer-storage-extension) | Provides interfaces for extending the data caching. |
+| MODULE                                                                                                      | DESCRIPTION                                                                                                 |
+|-------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| [ProductOfferStorage](https://github.com/spryker/product-offer-storage)                                     | Provides publisher plugins and operates data caching according to the entity changes                        |
+| [ProductOfferStorageExtension](https://github.com/spryker/product-offer-storage-extension)                  | Provides interfaces for extending the data caching.                                                         |
+| [MerchantProductOfferStorage](https://github.com/spryker/merchant-product-offer-storage)                    | Provides publisher plugins and operates product offer data caching according to the merchant entity changes |
```

Comment:
what about MerchantProductOfferStorageExtension? please update the diagram as well.

Reply:
`MerchantProductOfferStorageExtension` plugins are deprecated, I don't think that we should include it to the diagram